config/config.example.json

```diff
@@ -7,17 +7,20 @@
       "secret": "xxxxxxxx",
       "default_sgs": ["sg-xxxxxx", "sg-yyyyyy"],
       "default_subnets": ["subnet-xxxxxxx", "subnet-yyyyyy"],
-      "defaultExecutionRoleArn": "arn:aws:iam::1111111111111:role/ecsTaskExecutionRole"
+      "defaultExecutionRoleArn": "arn:aws:iam::1111111111111:role/ecsTaskExecutionRole",
+      "defaultKmsKeyId": "12121212-3333-4444-5555-676767676767"
     },
     "spinup": {
       "region": "us-east-1",
       "akid": "xxxxxxxx",
       "secret": "xxxxxxxx",
       "default_sgs": ["sg-zzzzzz"],
       "default_subnets": ["subnet-zzzzzz"],
-      "defaultExecutionRoleArn": "arn:aws:iam::22222222222222:role/ecsTaskExecutionRole"
+      "defaultExecutionRoleArn": "arn:aws:iam::22222222222222:role/ecsTaskExecutionRole",
+      "defaultKmsKeyId": "90909090-8888-7777-6666-545454545454"
```

Comment:
Not important, but why did we switch from snake case (`default_subnets`) to camel case here 😄

Reply:
too many devs in the code base? 😆